backend/src/main/java/ca/bc/gov/app/exception/NoFirstNationException.java

```diff
@@ -7,8 +7,12 @@
 
 @ResponseStatus(HttpStatus.NOT_FOUND)
 public class NoFirstNationException extends ResponseStatusException {
+
+  private static final long serialVersionUID = -2549888799966841467L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data

Reply:
Thanks. My IDE was complaining for that, so I added it :)